File: app/src/main/java/org/lemonadestand/btb/features/login/models/LoginResponse.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

public class LoginResponse {

    @SerializedName("status")
    boolean status;

    @SerializedName("message")
    String message;

    @SerializedName("user")
    User user;


    public void setStatus(boolean status) {
        this.status = status;
    }
    public boolean getStatus() {
        return status;
    }

    public void setMessage(String message) {
        this.message = message;
    }
    public String getMessage() {
        return message;
    }

    public void setUser(User user) {
        this.user = user;
    }
    public User getUser() {
        return user;
    }

}

File: app/src/main/java/org/lemonadestand/btb/models/LoginResponse.kt
package org.lemonadestand.btb.models

import com.google.gson.annotations.SerializedName

class LoginResponse {
	@SerializedName("status")
	var status: Boolean = false

	@SerializedName("message")
	var message: String? = null

	@SerializedName("user")
	var user: User? = null
}
